package.json

```diff
@@ -12,6 +12,8 @@
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.4.3",
     "react-scripts": "^5.0.1",
+    "redux": "^4.2.0",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/202355512-652031e5-04f1-41eb-a2b4-18d337c46a95.PNG)

- [ ] Your summary is not descriptive enough 😨 Kindly make it descriptive by clicking on the three dots. Add more details in your summary to make your pull request professional 👍